functions/Remove-DbaAgentOperator.ps1

```diff
@@ -49,42 +55,65 @@ function Remove-DbaAgentOperator {
 
         This removes an operator named DBA from the instance.
 
+    .EXAMPLE
+        PS C:\>  Get-DbaAgentOperator -SqlInstance SRV1 | Out-GridView -Title 'Select SQL Agent operator(s) to drop' -OutputMode Multiple | Remove-DbaAgentOperator
+
+        Using a pipeline this command gets all SQL Agent operator(s) on SRV1, lets the user select those to remove and then removes the selected SQL Agent alert category(-ies).
+
     #>
-    [CmdletBinding(DefaultParameterSetName = "Default", SupportsShouldProcess, ConfirmImpact = "High")]
```

Comment:
aw was there an issue with the defaultparametersetname?

Reply:
Not I am aware of, I re-used the template for other commands, so looks like this part is missing from all other commands that were recently created.

Reply:
Ah k. So if you can add back the defaultparametersetname then remove the "non-pipeline" parameterset, it'll be ready to merge 👍🏼